packages/xchain-cosmos/__e2e__/cosmos-client.e2e.ts

```diff
@@ -1,119 +1,96 @@
+      }
+    }),
+  }
+}
 
-describe('Cosmos Integration Tests', () => {
-  beforeEach(() => {
-    const settings = { network: Network.Testnet, phrase: process.env.TESTNET_PHRASE }
-    xchainClient = new CosmosClient(settings)
-  })
-  it('should fetch cosmos balances', async () => {
-    const address = await xchainClient.getAddressAsync(0)
-    const balances = await xchainClient.getBalance(address)
+describe('Cosmos client e2e', () => {
+  let client: Client
 
-    balances.forEach((bal) => {
-      console.log(`${address} ${assetToString(bal.asset)} = ${bal.amount.amount()}`)
+  beforeAll(() => {
+    client = new Client({
+      network: Network.Mainnet,
+      phrase: process.env.PHRASE_MAINNET,
     })
-    expect(balances.length).toBeGreaterThan(0)
   })
 
-  it('should generate cosmos addreses', async () => {
-    const address0 = await xchainClient.getAddressAsync(0)
-    const address1 = await xchainClient.getAddressAsync(1)
-
-    expect(address0.length).toBeGreaterThan(0)
-    expect(address0.startsWith('cosmos')).toBeTruthy()
-
-    expect(address1.length).toBeGreaterThan(0)
-    expect(address1.startsWith('cosmos')).toBeTruthy()
-  })
-  it('should transfer uatom from wallet 0 -> 1, with a memo', async () => {
-    try {
-      const addressTo = await xchainClient.getAddressAsync(2)
-      const transferTx: TxParams = {
-        walletIndex: 0,
-        asset: AssetATOM,
-        amount: baseAmount('100000', 6),
-        recipient: addressTo,
-        memo: 'Hi!',
-      }
-      const res = await xchainClient.transfer(transferTx)
-      expect(res.length).toBeGreaterThan(0)
-    } catch (error) {
-      throw error
-    }
+  it('Should get wallet address', async () => {
+    const address = await client.getAddressAsync()
+    console.log(address)
   })
-  it('should fail xfer 100000000000 from wallet 0 -> 1', async () => {
-    try {
-      const addressTo = await xchainClient.getAddressAsync(1)
-      const transferTx: TxParams = {
-        walletIndex: 0,
-        asset: AssetATOM,
-        amount: baseAmount('100000000000', 6),
-        recipient: addressTo,
-        memo: 'Hi!',
-      }
-      await delay(25 * 1000)
-      const txHash = await xchainClient.transfer(transferTx)
-      expect(txHash.length).toBeGreaterThan(0)
 
-      // Wait 35 seconds for the tx to process
-      await delay(15 * 1000)
-
-      const txResult = await xchainClient.getSDKClient().txsHashGet(txHash)
-      expect(txResult.raw_log?.includes('insufficient funds')).toBeTruthy()
-    } catch (error) {
-      throw error
-    }
+  it('Should get address balances', async () => {
+    const balances = await client.getBalance(await client.getAddressAsync())
+    console.log(
+      balances.map((balance) => {
+        return {
+          asset: assetToString(balance.asset),
+          amount: baseToAsset(balance.amount).amount().toString(),
+        }
+      }),
+    )
   })
-  it('Prepate transaction, sign externally and broadcast', async () => {
```

Comment:
Can we keep this test ? Some of the leading integrations use this system for signing because they do not work with private keys and it is their only option. Now that .getSDKClient() is not available, their way of signing will be different.